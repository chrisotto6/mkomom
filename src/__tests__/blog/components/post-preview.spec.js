import React from 'react'
import { create } from 'react-test-renderer'
import Preview from '../../../blog/components/post-preview'

describe('Preview', () => {
  it('renders correctly', () => {
    const tree = create(
      <Preview
        fluidImage="test img"
        slug="test slug"
        title="test title"
        date="10-06-1988"
        category="test"
        description="test description"
      />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
