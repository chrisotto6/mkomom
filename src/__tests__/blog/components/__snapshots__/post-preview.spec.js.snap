// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preview renders correctly 1`] = `
<div
  className="pv3 flex justify-center items-center flex-wrap"
>
  <div
    className="w-100 mw6 h-100 gatsby-image-wrapper"
    style={
      Object {
        "maxHeight": null,
        "maxWidth": null,
        "overflow": "hidden",
        "position": "relative",
      }
    }
  >
    <div
      aria-hidden={true}
      style={
        Object {
          "paddingBottom": "NaN%",
          "width": "100%",
        }
      }
    />
    <picture>
      <img
        alt=""
        loading="lazy"
        onLoad={[Function]}
        style={
          Object {
            "height": "100%",
            "left": 0,
            "objectFit": "cover",
            "objectPosition": "center",
            "opacity": 0,
            "position": "absolute",
            "top": 0,
            "transition": "opacity 500ms",
            "width": "100%",
          }
        }
      />
    </picture>
    <noscript
      dangerouslySetInnerHTML={
        Object {
          "__html": "<picture><source srcset=\\"undefined\\" /><img loading=\\"lazy\\" src=\\"\\" alt=\\"\\" style=\\"position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\\"/></picture>",
        }
      }
    />
  </div>
  <div
    className="mw7 pa2 ph4-ns"
  >
    <span
      className="db f2 display"
    >
      <a
        className="dark-gray no-underline"
        href="test slug"
      >
        test title
      </a>
    </span>
    <div
      className="mv3 mb5-ns flex justify-between"
    >
      <div
        className="db f6 silver ttu tracked sans-serif"
      >
        10-06-1988
      </div>
      <div
        className="db f6 silver ttu tracked sans-serif"
      >
        TAGGED: 
        test
      </div>
    </div>
    <div
      className="serif f4 lh-copy"
    >
      test description
    </div>
  </div>
</div>
`;
